az-hadoop-jobtype-plugin/src/main/java/azkaban/jobtype/HadoopJobUtils.java

```diff
@@ -80,11 +84,20 @@ public class HadoopJobUtils {
   public static final String MAPREDUCE_JOB_OTHER_NAMENODES = "mapreduce.job.hdfs-servers";
   // MapReduce config for mapreduce job tags
   public static final String MAPREDUCE_JOB_TAGS = "mapreduce.job.tags";
+
+  public static final String YARN_KILL_VERSION = "yarn.kill.version";
+  // values of the yarn kill version flag
+  public static final String YARN_KILL_LEGACY = "legacy";
+  public static final String YARN_KILL_TOKEN_FIX = "token_copy";
```

Comment:
lets rename to YARN_KILL_USING_API

Reply:
@tsangz2013  please address this. token copy does not sound like the right way.

Reply:
updated the naming to `api_with_token`